Puesdo Hooks/XComStrategyGame/XGStrategySoldier/HasPerk.txt

```diff
@@ -19,10 +19,10 @@ function bool HasPerk(int iPerk)
 }
 */
 
-	19 10 25 19 2E <XComGame.XComGameInfo> 19 01 <Engine.Actor.WorldInfo> [@] <Engine.WorldInfo.Game> 00 ( 01 <Engine.WorldInfo.Game> ) [@] <XComGame.XComGameInfo.Mods> 00 ( 01 <XComGame.XComGameInfo.Mods> ) [@] <NullRef> 00 ( 1B <Object> 01 <@m_kChar> 16 ) 16
+	19 19 10 25 19 2E <Class.XComGameInfo> 19 01 <Engine.Actor.WorldInfo> [@] <Engine.WorldInfo.Game> 00 ( 01 <Engine.WorldInfo.Game> ) [@] <XComGameInfo.Mods> 00 ( 01 <XComGameInfo.Mods> ) [@] <NullRef> 00 ( 1B <Mods> 1F <%t "RoulettePlus.RoulettePlusMod"> 16 ) [@] <NullRef> 00 ( 1B <Object> 17 27 16 )
```

Comment:
you've changed the package reference for XComGame as if it were in XComGame, but it is not.
ie: `2E <XComGame.XComGameInfo>` vs `2E <Class.XComGameInfo>`